protocol/lavasession/consumer_session_manager.go

```diff
@@ -20,6 +20,14 @@ const (
 	debug = false
 )
 
+type SessionInfo struct {
```

Comment:
move to consumer_types

Reply:
Moved 6dfcd732f31c68124804711c5a62cfefacce56e7